dsaas-services/rhche-redirector.yaml

```diff
@@ -0,0 +1,6 @@
+services:
+- hash: master
```

Comment:
do we really want to use master instead of some particular commit-id?

Reply:
We had been using master for the other redirector. If this isn't a problem then lets leave it at master for now